### PR TITLE
added q parameter to the List Users Assigned to an Application table

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
Added q parameter to the List Users Assigned to an Application table

### Resolves:

* [OKTA-101378](https://oktainc.atlassian.net/browse/OKTA-101378)

### CODE REVIEW

File: _source/_docs/api/resources/apps.md [hunk 1]
```diff
@@ -3212,6 +3212,7 @@ Parameter | Description                                                      | P
 applicationId       | `id` of an [app](#application-model)                   | URL        | String   | TRUE     |
 limit     | Specifies the number of results per page (maximum 500)           | Query      | Number   | FALSE    | 50
 after     | Specifies the pagination cursor for the next page of assignments | Query      | String   | FALSE    |
+q         | Returns a filtered list of app users. The value of `q` is only matched against a user's app `userName`. **Note:** This operation only supports `startsWith` semantics. | Query      | String   | FALSE    |
```
Comment (williamyang-okta) [inline _source/_docs/api/resources/apps.md]: it will match against OKTA user's firstName or lastName or email

Comment (susanharper-okta) [reply _source/_docs/api/resources/apps.md]: @williamyang-okta & @tthompson-okta 

As I understand it (and how it seemed to work for me in Postman), the List AppUsers API **only supports** search by username (meaning the **appusername**).

Comment (williamyang-okta) [reply _source/_docs/api/resources/apps.md]: @susanharper-okta  verified the logic and the code, you are right,  it search through 
appUser's  userName, firstName, lastName and email
But most no provision app only has the username

Comment (susanharper-okta) [reply _source/_docs/api/resources/apps.md]: @williamyang-okta - I have updated the text per our discussion yesterday. Can you please review? Thank you.

File: _source/_docs/api/resources/apps.md [hunk 2]
```diff
@@ -3212,6 +3212,7 @@ Parameter | Description                                                      | P
 applicationId       | `id` of an [app](#application-model)                   | URL        | String   | TRUE     |
 limit     | Specifies the number of results per page (maximum 500)           | Query      | Number   | FALSE    | 50
 after     | Specifies the pagination cursor for the next page of assignments | Query      | String   | FALSE    |
+q         | Returns a filtered list of app users. The value of `q` is matched against a user's app `userName`, `firstName`, `lastName` and `email`. **Note:** This operation only supports `startsWith` semantics. | Query      | String   | FALSE    |
```
Comment (jakubvul) [inline _source/_docs/api/resources/apps.md]: I think we need to explain a bit more what that note means. Perhaps you can add more information below the table.

**Comment by susanharper-okta** [comment]
@jakubvul  - made updates per my discussion with William today. Can you please review?

**Pull request merged**